feat: add February FGN (#970)
openfga/openfga.dev: blog/fine-grained-news-2025-02.md

---
title: Fine Grained News - February 2025
description: Fine Grained News
slug: fine-grained-news-2025-02
date: 2025-02-26
authors: aaguiar
tags: [newsletter]
image: https://openfga.dev/img/og-rich-embed.png
hide_table_of_contents: false
---
# Fine Grained News - February 2025

Welcome to the second Fine Grained News edition of 2025! 

## Just Shipped!

- We shipped 3 minor versions of OpenFGA which include:

>   - A fix for [CVE-2025-25196](https://github.com/openfga/openfga/security/advisories/GHSA-g4v5-6f5p-m38j) and [CVE-2024-56323](https://github.com/openfga/openfga/security/advisories/GHSA-32q6-rr98-cjqv)

Reviewer: ```suggestion
  - Fixes for [CVE-2025-25196](https://github.com/openfga/openfga/security/advisories/GHSA-g4v5-6f5p-m38j) and [CVE-2024-56323](https://github.com/openfga/openfga/security/advisories/GHSA-32q6-rr98-cjqv)
```

  - Several performance improvements that are enabled with the `enable-check-optimizations` experimental flag.
  - Dynamic TLS certificate reloading for HTTP and gRPC servers. Thanks [Rokibul Hasan](https://github.com/RokibulHasan7) for your contribution!
  - A `name` filter to ListStores. The name parameter instructs the API to only include results that match that name. Thanks [Karl Persson](https://github.com/kalleep) for your contribution!
  - Optimized database dialect handling by setting it during initialization instead of per-call, fixing SQL syntax errors in MySQL. Thanks [Siddhant Khare](https://github.com/Siddhant-K-code) for your contribution!

> 

Reviewer: There seems to be something off in the new lines and the indentation of the bullet points

Author: They look good on the web page

  - Support for Go 1.24. We follow Go's version support policy and will only support the latest two major versions of Go. Now that Go 1.24 is out, we have dropped support for Go < 1.23.
  

> - Two minor versions of the [Java SDK](https://github.com/openfga/java-sdk), with support for Batch Check. Thanks [Piotr Olaszewski](https://github.com/piotrooo) for your contribution!

Reviewer: ```suggestion
- Two minor versions of the [Java SDK](https://github.com/openfga/java-sdk), with support for server-side [BatchCheck](https://github.com/openfga/js-sdk?tab=readme-ov-file#batch-check). Thanks [Piotr Olaszewski](https://github.com/piotrooo) for your contribution!
```

> - A minor release of the [Go SDK](https://github.com/openfga/go-sdk), with support for the `start-time` parameter in the Changes API and support for specifying contextual tuples and context parameters in assertions.

Reviewer: ```suggestion
- A minor release of the [Go SDK](https://github.com/openfga/go-sdk), with support for the `StartTime` parameter in the [ReadChanges method](https://github.com/openfga/go-sdk?tab=readme-ov-file#read-relationship-tuple-changes-watch) and support for specifying contextual tuples and context parameters in assertions.
```

<!-- markdown-link-check-enable -->

- [Poovamraj Thanganadar Thiagarajan](https://www.linkedin.com/in/poovamraj/) from Okta will be presenting at KubeCon Europe, together with [Jo Guerreiro](https://www.linkedin.com/in/jmlguerreiro/) from Grafana Labs about [From Chaos To Control: Migrating Access Control To OpenFGA in a Multi-Tenant World](https://kccnceu2025.sched.com/event/1txIJ/from-chaos-to-control-migrating-access-control-to-openfga-in-a-multi-tenant-world-jo-guerreiro-grafana-labs-poovamraj-thanganadar-thiagarajan-okta).

- [Andres Aguiar](https://www.linkedin.com/in/aaguiar/) from Okta was invited to present on the Maintainer's Summit at KubeCon Europe about our experiences collaborating with the CNCF TAG-Security team: [A Project Maintainers Guide To TAG Security](https://maintainersummiteu2025.sched.com/event/1tj8v/a-project-maintainers-guide-to-tag-security-marina-moore-edera-andres-aguiar-okta).

## **See You Next Month:**

> Fine Grained News is published every month. If you have any feedback, want to share your OpenFGA story, or have a noteworthy update, please let us know on any of our [community channels](https://openfga.dev/community) or at [community@openfga.dev](mailto:community@openfga.dev).

Reviewer: Instead of putting the email here, we should add it to the community page itself

Author: It does not hurt to have it here.. we can add it to the page too
